source/get-started/setup/index.md

```diff
@@ -3,186 +3,116 @@ title: Setting Up PatternFly
-  </p>
-  <p>As part of this "how to", we are using MacOS running El Capitan.</p>
-</div>
 <div class="row">
   <div class="col-md-12">
-    <h2>Prerequisites</h2>
-    <div class="row">
-      <div class="col-xs-12 col-sm-12 col-md-12">
-        <p>Before you can use PatternFly, you must have the following utilities installed:</p>
-        <ul>
-          <li><a href="https://git-scm.com/downloads" target="top">Git</a>: a free and open source distributed version control system.
-            <ul>
-              <li>Unfamiliar with Git? Try <a href="https://desktop.github.com/" target="top">GitHub for Windows</a> or <a href="https://desktop.github.com/" target="top">GitHub for Mac</a></li>
-            </ul>
-          </li>
-          <li><a href="https://nodejs.org/en/" target="top">NodeJS (v4.x LTS)</a>: a software platform that is used to build server-side applications.
-            <ul>
-              <li>
-                <kbd>$ sudo gem install node</kbd>
-              </li>
-            </ul>
-          </li>
-          <li><a href="http://gruntjs.com/getting-started" target="top">Grunt</a>: a JavaScript task runner.
-            <ul>
-              <li>
-                <kbd>$ sudo npm install -g grunt-cli</kbd>
-              </li>
-            </ul>
-          </li>
-          <li><a href="https://bower.io/#installing-bower" target="top">Bower</a>: a package manager for the web.
-            <ul>
-              <li>
-                <kbd>$ sudo npm install -g bower</kbd>
-              </li>
-            </ul>
-          </li>
-        </ul>
-      </div>
-    </div>
+    <h2>Directly introduce PatternFly with CDN provider</h2>
   </div>
 </div>
 <div class="row">
   <div class="col-md-12">
-    <h2>Step 1: Create A Project Directory</h2>
-    <div class="row">
-      <div class="col-xs-12 col-sm-12 col-md-6">
-        <h3>Using Finder</h3>
-        <ul>
-          <li>Launch a new Finder window</li>
-          <li>Navigate to where you want to project to live</li>
-          <li>File -> New Folder (or Ctrl + Shift + N)</li>
-          <li>Name the folder</li>
-        </ul>
-      </div>
-      <div class="col-xs-12 col-sm-12 col-md-6">
-        <h3>Using Command Line</h3>
-        <p>
-          <ul>
-            <li>Open Terminal</li>
-            <li>Navigate to where you want the folder<br />
-              <kbd>
-                cd "path_to/projects"
-              </kbd>
-            </li>
-            <li>Create the directory with the project name<br/>
-              <kbd>
-                mkdir myProject
-              </kbd>
-            </li>
-          </ul>
-        </p>
-      </div>
+    <p>
+      If you want to try out the components provided by PatternFly in a fast way. Just use these <a href="https://cdnjs.com/libraries/patternfly" target="top">PatternFly</a> CDN links.
```

Comment:
We don't really publish anything on cdnjs, if we want to recommend using a CDN I think we should consider using rawgit CDN or start maintaining a CDN ourselves.